volatility3/cli/volshell/__init__.py

```diff
@@ -108,10 +118,33 @@ def run(self):
         os_specific.add_argument("-l", "--linux", default = False, action = "store_true", help = "Run a Linux volshell")
         os_specific.add_argument("-m", "--mac", default = False, action = "store_true", help = "Run a Mac volshell")
 
+        parser.set_defaults(**default_config)
+
         # We have to filter out help, otherwise parse_known_args will trigger the help message before having
         # processed the plugin choice or had the plugin subparser added.
         known_args = [arg for arg in sys.argv if arg != '--help' and arg != '-h']
         partial_args, _ = parser.parse_known_args(known_args)
+
+        ### Start up logging
+        if partial_args.log:
+            file_logger = logging.FileHandler(partial_args.log)
+            file_logger.setLevel(1)
+            file_formatter = logging.Formatter(datefmt = '%y-%m-%d %H:%M:%S',
+                                               fmt = '%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
+            file_logger.setFormatter(file_formatter)
+            rootlog.addHandler(file_logger)
```

Comment:
The `rootlog` variable does not appear to exist in that source file.
```python
rootlog = logging.getLogger()
```

Reply:
Thanks, I didn't spot I pulled that over, but it should now be resolved in `commit 9ea1a96`

Reply:
Thank you for apply! :)